app.py

```diff
@@ -39,36 +39,25 @@
     cdk_app, f"{stack_name_prefix}-load-balancer", network_stack.vpc
 )
 
-apex_service_props = ServiceProps(
-    "schematic-apex",
-    8000,
-    200,
-    f"ghcr.io/sage-bionetworks/schematic-apex:{image_version}",
-    {
-        "API_DOCS_HOST": "schematic-api-docs",
-        "API_DOCS_PORT": "8010",
-        "API_GATEWAY_HOST": "schematic-api-gateway",
-        "API_GATEWAY_PORT": "8082",
-        "APP_HOST": "schematic-app",
-        "APP_PORT": "4200",
-        "THUMBOR_HOST": "schematic-thumbor",
-        "THUMBOR_PORT": "8889",
-        "ZIPKIN_HOST": "schematic-zipkin",
-        "ZIPKIN_PORT": "9411",
-    },
+app_service_props = ServiceProps(
+    "schematic-app",
+    443,
+    4096,
+    "ghcr.io/sage-bionetworks/schematic:v0.1.90-beta",
+    {},
+    "schematic-dev-DockerFargateStack/dev/ecs",
 )
 
 app_service_stack = LoadBalancedServiceStack(
     cdk_app,
     f"{stack_name_prefix}-app",
     network_stack.vpc,
     ecs_stack.cluster,
-    apex_service_props,
+    app_service_props,
```

Comment:
To help @BryanFauble you might add the second service, just running a 'dummy' container like [hello-world](https://hub.docker.com/_/hello-world), to demonstrate how that is added.

Reply:
Also to mention that there will be more than a single port for the otel collector that will be used. The healthcheck endpoint, and the port to send telemetry data is another.

Is there are examples of that I would be really grateful to see them

Reply:
> To help @BryanFauble you might add the second service, just running a 'dummy' container like hello-world, to demonstrate how that is added.

I'm not a fan of adding unnecessary code.  Instead i'll point to the [OC infra app.py](https://github.com/Sage-Bionetworks-IT/openchallenges/blob/dev/app.py) for an example of adding additional containers to the deployment.

> Also to mention that there will be more than a single port for the otel collector that will be used. The healthcheck endpoint, and the port to send telemetry data is another.

Opening multiple ports to a container would be a slight variation that is not done in the OC setup.   There are many different ways to deal with that but I think it would be best to move forward with this PR then determine how we should incorporate the OTEL collector.